e2e/bash_unit

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env bash
```

Comment:
we don't need to vendor this file since it gets downloaded by the `$(BASH_UNIT)` Makefile target, right?

Reply:
yes

Reply:
I think removing this file is the only thing left to merge this PR

Reply:
yes i forgot to `git rm` it